### PR TITLE
Cookieconsent next

### PR DESCRIPTION
**Pull request by kptdobe**
Included some feedback from various people.

Test URLs:
- Before: https://main--aem-block-collection--adobe.hlx.page/block-collection/cookie-consent-details-demo
- After: https://cookieconsent-next--aem-block-collection--adobe.hlx.page/block-collection/cookie-consent-details-demo

### CODE REVIEW

File: blocks/consent/consent.js [hunk 1]
```diff
@@ -82,3 +84,19 @@ export function showConsentForUpdate(consentName) {
...
   import('./consent-banner.js').then((ccdialog) => ccdialog.showConsentBannerForUpdate(consentName, manageConsentUpdate));
 }
+
+/**
+ * Updates the Cookie Preference link inside the provided element:
+ * on click, the link will open the consent dialog
+ * if the 'cookie-consent' metadata is set on the page.
+ * @param {Element} el the DOM element in which the consent preference link is to be set up
+ */
+export function setupConsentPreferenceLink(el = document) {
```
Comment (chicharr) [inline blocks/consent/consent.js:94]: this is a very good idea

File: blocks/consent/README.md [hunk 2]
```diff
@@ -57,16 +57,23 @@ The second and subsequent sections are considered to be cookie categories. And e
...
 
 * In case of *#1. Simple consent banner* this function will show the minimalistic non-intrusive banner
 * In case of *#2. Simple consent banner and categories details* this function would show the consent categories detail dialog.
+
+## Block setup
+
+Block needs to be loaded as quickly as possible and the logic to load the block or not highly depends on project needs. Here are the two elements you may want to patch in your project:
+
+- in [scripts.js](../../scripts.js), you need to load the consent block in the lazy phase to load the consent banner
+- in [header](../header/header.js), you can "patch" the Cookie Preferences link to open the consent dialog using the `setupConsentPreferenceLink` function from the block.
```
Comment (chicharr) [inline blocks/consent/README.md:79]: isn't this footer?

**Review by chicharr — APPROVED** [review]
LGTM. Just a comment regarding the changes in the doc

File: blocks/consent/README.md (hunk 2 above)
Comment (kptdobe) [reply blocks/consent/README.md]: of course!

**Pull request merged**